src/docs/testing/best-practices.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Performance best practices
+short-title: Best practices
+description: How to ensure that your Flutter app is performant.
+---
+
+Generally Flutter applications are high performance by default,
```

Comment:
"are" high performance or "have" high performance?

"Apps are performance" sounds a little strange to me. But feel free to ignore this if it's actually a common use case in English (My English is very limited...)

Reply:
`are performant`?

Reply:
Sounds good.